tests/unit/dom-event-listeners-test.js

```diff
@@ -557,37 +531,4 @@ module('ember-lifeline/dom-event-listeners', function (hooks) {
     await triggerEvent(window, 'click');
     assert.equal(calls, 1, 'callback was called');
   });
-
-  test('runDisposables more than once', async function (assert) {
```

Comment:
I think we need to bring this test back and add another test for calling runDestroyables _while_ the native / platform destroyables are running.

Reply:
As per my previous comment. We're removing this feature (running `runDestroyables` multiple times).

Reply:
Looking at this a bit closer - can you explain your rationale for wanting these tests? The reason I ask is that, without any disposables being registered for a given object, the `runDisposables` call would early return. What are we trying to test here?

I still believe that adding this specific test - running disposables more than once - should be added to the core of the disposables utility tests.

Reply:
Synced offline. We determined the correct number of tests we need.